pkg/filesystem/subvolume_test.go

```diff
@@ -59,3 +63,34 @@ func TestGetOmapVal(t *testing.T) {
 		})
 	}
 }
+
+func TestGetSubvolumeNameFromPath(t *testing.T) {
+
+	tests := []struct {
+		path string
+		name string
+	}{
+		{
+			path: "/volumes/csi/csi-vol-6a99b552-fdcc-441d-b1e6-a522a85a503d/5f4e4caa-f835-41ba-83c1-5bbd57f6aedf",
+			name: "csi-vol-6a99b552-fdcc-441d-b1e6-a522a85a503d",
+		},
+		{
+			path: "",
+			name: "",
+		},
+		{
+			path: "/volumes/csi-vol-6a99b552-fdcc-441d-b1e6-a522a85a503d/5f4e4caa-f835-41ba-83c1-5bbd57f6aedf",
```

Comment:
Does this unit test pass? If there are fewer than 4 slashes, it's supposed to return the empty string, so I expect it to fail. 
@subhamkrai Are we running the unit tests in the CI?

Reply:
No it will not fail, the extra slash at initial adds an empty string as first member of list, hence the total length is still 4, and due to this it is not failing. @travisn

Reply:
oh of course, my error